src/chargemaster_parsers/parsers/uci.py

```diff
@@ -0,0 +1,53 @@
+from . import ChargeMasterEntry, ChargeMasterParser
+import json
+
+class UCIChargeMasterParser(ChargeMasterParser):
+        INSTITUTION_NAME = "UCI"
+        ARTIFACT_URL = "https://www.ucihealth.org/-/media/files/xlsx/patients-visitors/952226406-regentsoftheuniversityofcaliforniaatirvinehospital-standardcharges.json?la=en&hash=0C51B7235E101EB9E6FA133A2D3AC5C7F297EB5C"
+        ARTIFACT_URLS = (ARTIFACT_URL, )
+
+        def parse_artifacts(self, artifacts):
+            for artifact_url, artifact in artifacts.items():
+                data_dict = json.load(artifact)
+                
+
+                for category, entries in data_dict.items():
+                    # Currently only parsing the 'Gross Charges' entries. This could be extended to parse the 
+                    # other cateogories, such as 'Outpatient De-identified Negotiated Charge', but would require 
+                    # some more research.
+                    if category == 'Gross Charges':
```

Comment:
This format is actually quite like Stanford which I'm looking into now. Kind of sad they only include insurance information for inpatient MS-DRG's but such is life.

Also interesting that they were careful to only use HCPCS codes not CPT extensions.